Front-End/src/Pages.js

```diff
@@ -392,12 +406,14 @@ function Intake(props) {
                         <option value="ONYX">ONYX</option>
                     </select>
                     <br></br>
-                    <input type="text" id="LengthInInches" name="LengthInInches" placeholder="Length in Inches" className="text-input" required="required" />
+
+                    <input type="text" id="LengthInInches" name="intake_LengthInInches" placeholder="Length in Inches" className="text-input" required="required" />
                     <br></br>
-                    <input type="text" id="PartNumber" name="PartNumber" placeholder="Part Number" className="text-input" required="required" />
+                    <input type="text" id="PartNumber" name="intake_PartNumber" placeholder="Part Number" className="text-input"  required="required" />
                     <br></br>
                     <h4>When do you need the part done by?</h4>
-                    <NewDate />
+                    <NewDate name="intake_date" />
+                    {/* <label for="avatar">Choose a profile picture:</label>
```

Comment:
It looks like everything is commented out now, and it's causing the builds to fail. We probably need to fix this up before merging in